openpower/package/hostboot/hostboot.mk

```diff
@@ -13,8 +13,15 @@ HOSTBOOT_DEPENDENCIES = host-binutils
 HOSTBOOT_INSTALL_IMAGES = YES
 HOSTBOOT_INSTALL_TARGET = NO
 
+# The BR2_HOSTBOOT_CONFIG_FILE variable has quotes around it, which throws off the wildcard function
+BR2_HOSTBOOT_CONFIG_FILE_NONQUOTED := $(shell echo $(BR2_HOSTBOOT_CONFIG_FILE))
```

Comment:
BR2_HOSTBOOT_CONFIG_FILE_NONQUOTED := $(subst ",,$(BR2_HOSTBOOT_CONFIG_FILE))

Using shell+echo here is dangerous.  Other than some unintended consequences if someone were to put &&, ||, or ; into the variable, it has the side effect of evaluating backslash escaped characters, which causes them to lose their backslash.